src/schemathesis/runner/impl/solo.py

```diff
@@ -1,6 +1,6 @@
 # weird mypy bug with imports
```

Comment:
If the new mypy version won't consider removing the imports below as a bug, then this line could be removed as well

Reply:
Fixed.